src/ticketmap/index.tsx

```diff
@@ -327,39 +339,49 @@ export default class TicketMap extends Component<Props, State> {
    * Public Methods
    */
 
-  highlightSection = (section: string) => this.toggleSectionHighlight(section, true)
+  highlightSection = (section: string): void => this.toggleSectionHighlight(section, true)
 
-  unhighlightSection = (section: string) => this.toggleSectionHighlight(section, false)
+  unhighlightSection = (section?: string): void => {
+    if (!section) {
+      this.setState({ currentHoveredSection: null })
+      return this.updateMap()
+    }
+    return this.toggleSectionHighlight(section, false)
+  }
 
-  toggleSectionHighlight = (section: string, shouldHighlight = true) => {
+  toggleSectionHighlight = (section?: string, shouldHighlight: boolean = true): void => {
     if (!section) {
       return
     }
-    const sectionId = section.toLowerCase()
-    const isUnhighlightingSelectedSection = !shouldHighlight && this.state.selectedSections.has(sectionId)
+
+    const sectionId: string = section.toLowerCase()
```

Comment:
This type hint seems unnecessary IMO. `string.toLowerCase()` is already strictly typed.

Reply:
```suggestion
    const sectionId = section.toLowerCase()
```